### ISSUE TITLE
ErrorDisplay accessibility

### ISSUE THREAD

**Issue by qroll** [LifeSG/react-design-system#883]
**Component**
`ErrorDisplay`

**Describe the feature request**
- Align ErrorDisplay with the accessibility behaviour defined in the Figma specs
- Set the heading element to render as `h2` (independent of styling)
- Set empty alt for the illustration

**Acceptance criteria**
- Roles should be correctly read out by the AT
- Minimally test in NVDA + Chrome (Windows), VO + Safari (macOS), Talkback + Chrome (Android)

**References**
[Figma](https://www.figma.com/design/wDAxbRmlq4xOn4UJ8zPuEF/%F0%9F%A7%91%F0%9F%8F%BB%E2%80%8D%F0%9F%A6%AF-Flagship-A11y-Annotations?node-id=2496-1594&p=f&m=dev)
